src/graphic.tex

```diff
@@ -244,22 +267,24 @@ \section{Curves and Shapes}
 \tikzset{baseline} \vspace{-1cm} %!hide
 \tikz{\draw (0, 0)
   to[out=90, in=-90, looseness=0.5] (2, 0);}
-\tikz{\draw (0, 0) 
+\tikz{\draw (0, 0)
   to[out=90, in=-90, looseness=2] (2, 0);}
 \vspace{-1cm} %!hide
 \end{example}
 Often it might be easier to specify relative angles, using the \cargv{bend
-  left} and \cargv{bend right} keys. If no angle is provided a default value is
+  left} and \cargv{bend right} keys. If no angle is provided, a default value is
 used.
 \begin{example}[vertical_mode, examplewidth=0.8\linewidth]
 \tikz{\draw (0, 0) to[bend left] (2, 0);}
 \tikz{\draw (0, 0) to[bend right=90] (2, 0);}
 \end{example}
 
-If you need even finer control over the curves you can use a
-\ltx{.. controls ..} connection for specifying a Bézier curve with one
+If you need even finer control over the curves you, can use a
+\ltx{.. controls ..} connection to specify a Bézier curve with one
+%$ Bézier seems to work (I had no idea). Should we be using B\'ezier, or is that
+%$ so thirty years ago?
```

Comment:
```suggestion
%$ Bézier seems to work (I had no idea). Should we be using B\'ezier, or is that
%$ so thirty years ago?
```
Since we are all utf8 now, this is no problem, it was even ok back in latin1 days.